journal/scripts/load_indexedat_xlsx.py

```diff
@@ -0,0 +1,12 @@
+import openpyxl
+from ..models import IndexedAt
+
+
+def run():
+    xlsx_file_path = 'journal/fixtures/INDEX_AT.xlsx'
+    workbook = openpyxl.load_workbook(xlsx_file_path)
```

Comment:
Trocar
```python
def run():
    xlsx_file_path = 'journal/fixtures/INDEX_AT.xlsx'
```
por 
```python
def run(xlsx_file_path):
```
O arquivo journal/fixtures/INDEX_AT.xlsx não deve ser versionado, exceto ser for uma amostra, um modelo com poucos dados